.travis.yml

```diff
@@ -42,8 +42,8 @@ install:
     - if [ "$deps" = "no" ]; then composer --prefer-source install; fi;
 
 script:
-    - components=`find src/Symfony -mindepth 3 -type f -name phpunit.xml.dist -print0 | sed 's#\\(.*\\)/.*#\\1#'`
-    - if [ "$deps" = "no" ]; then echo $components | parallel -0 --gnu --keep-order 'echo -e "\\nRunning {} tests"; phpunit --exclude-group tty,benchmark,intl-data {} || (echo -e "\\e[41mKO\\e[0m {}" && $(exit 1));'; fi;
+    - components=$(find src/Symfony -mindepth 3 -type f -name phpunit.xml.dist -printf '%h\n')
```

Comment:
@nicolas-grekas for PRs changing the Travis config, merging them before the end of the Travis build is always too fast